frontend/src/components/LoadData.tsx

```diff
@@ -478,6 +484,44 @@ export function LoadData(props: { children?: ReactNode }) {
     }
   }, [setSettings, setters, setLoaded])
 
+  // Query for GlobalHub check
+  const globalHubQueryFn = useCallback(() => {
+    return getRequest<{
+      isGlobalHub: boolean
+    }>(getBackendUrl() + '/globalhub')
+  }, [])
+  const {
+    data: globalHubRes,
+    loading: globalHubLoading,
+    startPolling: globalHubStartPoll,
+    stopPolling: globalHubStopPoll,
+  } = useQuery(globalHubQueryFn, [{ isGlobalHub: false }])
+
+  // Start all Polls for Global values here
+  useEffect(() => {
+    globalHubStartPoll()
+    return () => {
+      // Stop polls on dismount
+      globalHubStopPoll()
+    }
+  }, [globalHubStartPoll, globalHubStopPoll])
+
+  // Update global value setters when data has finished
+  // if all global value queriees have finished set setGlobalValuesLoaded to true
+  useEffect(() => {
+    if (globalHubRes && !globalHubLoading) {
+      setIsGlobalHub(globalHubRes[0].isGlobalHub)
+      setGlobalValuesLoaded(true)
+    }
+  }, [globalHubRes, globalHubLoading, setIsGlobalHub])
+
+  // If all data not loaded (!loaded) & events data is loaded (eventsLoaded) && global values is loaded (globalValuesLoaded) -> set loaded to true
+  useEffect(() => {
+    if (!loaded && eventsLoaded && globalValuesLoaded) {
+      setLoaded(true)
+    }
+  }, [loaded, setLoaded, eventsLoaded, globalValuesLoaded])
+
```

Comment:
These 2 should not need to be inside `useEffect`. 

Did you need to introduce `globalValuesLoaded` state? I think you can just use `loading` from the `useQuery`.

Reply:
In general, you can update state without being inside an effect. The current render will be abandoned and restarted with the new state. This is more efficient than an effect, which will only happen after the full render completes.

https://react.dev/learn/you-might-not-need-an-effect

Reply:
I removed the useEffect from the 2nd instance mentioned above. Removing useEffect from the first instance causes too many rerenders.

I included `globalValuesLoaded` thinking it could be used were we to introduce other global values in the future. With only the global hub values being used now I can remove the state and use loading from useQuery.

Reply:
Was able to remove the effect now that globalValuesLoaded was removed.